city_scrapers/spiders/ind_indygo_gov_audit.py

```diff
@@ -17,84 +31,35 @@ def parse(self, response):
         Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
         needs.
         """
-        for item in response.css(".content-section:nth-child(8) ul li"):
+        meeting_section = response.css(".content-section:nth-child(8)")
+        section_title = meeting_section.css("h2::text").get()
+        meeting_year = section_title.split(" ")[0]
+        meeting_time = (
+            (response.css(".content-section:nth-child(8) p strong::text").get())
+            .split(":", 1)[1]
+            .strip()
+        )
```

Comment:
Same this [comment](https://github.com/City-Bureau/city-scrapers-indianapolis/pull/14/files#r1475200669).